src/routes/(blog-article)/deploying-torrust-to-production/+page.md

```diff
@@ -0,0 +1,1073 @@
+
+<CodeBlock lang="bash">
+
+```bash
+sudo ufw default deny incoming
+sudo ufw default allow outgoing
+sudo ufw allow ssh
+sudo ufw allow 443
+sudo ufw allow 6969/udp
+sudo ufw enable
+sudo ufw status verbose
+```
+
+</CodeBlock>
+
+<Callout type="info">
+  However, there is a problem with setting up your own firewall. By default, Docker manipulates iptables to set up network isolation. However, this can interfere with UFW. To fix this, you need to tell Docker not to modify iptables.
+</Callout>
+
+### Health checks
+
+Some services expose public health checks endpoints.
+
+Via proxy (HTTP):
+
+- Index API: <http://index.torrust-demo.com/api/health_check>
+- Tracker API: <http://tracker.torrust-demo.com/api/health_check>
+- Tracker HTTP: <http://tracker.torrust-demo.com/health_check>
+
+Via proxy (HTTPs):
+
+- Index: <https://index.torrust-demo.com/api/health_check>
+- Tracker API: <https://tracker.torrust-demo.com/api/health_check>
+- Tracker HTTPs: <https://tracker.torrust-demo.com/health_check>
+
+Direct access:
+
+- Index API: <http://tracker.torrust-demo.com:3001/health_check>
+- Tracker API: <http://tracker.torrust-demo.com:1212/health_check>
+- Tracker HTTP: <http://tracker.torrust-demo.com:7070/health_check>
+
+<Callout type="info">
+  These endpoints are very useful to test if your installation is OK.
+</Callout>
+
+## Troubleshooting
+
+You might have problems setting upt the application. There are some useful things you should know
+in order to be able to address potential problems.
+
+First, it might be useful to enable the debug logging for some services. Both the Index and the Tracker
+have a configuration option `log_level = "info"`. You can change the logging level to `debug` or even `trace`.
+
+Configuration files are located in:
+
+- `./storage/index/etc/index.toml`.
+- `./storage/tracker/etc/tracker.toml`.
+
+If you change the config files you will need to restart the services:
+
+<CodeBlock lang="bash">
+
+```bash
+docker compose restart
+```
+
+</CodeBlock>
+
+You can also check the logs of any service with:
+
+If you change the config files you will need to restart the services:
+
+<CodeBlock lang="bash">
+
+```bash
+docker compose logs proxy
+docker compose logs certbot
+docker compose logs index
+docker compose logs index-gui
+docker compose logs tracker
+```
+
+</CodeBlock>
+
+You can also enable the debug for the Nginx proxy
+
+Edit the file `./storage/proxy/etc/nginx-conf/nginx.conf` and add `error_log` to the server configuration:
+
+<CodeBlock lang="nginx">
+
+```nginx
+server
+{
+    listen 443 ssl http2;
+    listen [::]:443 ssl http2;
+    server_name index.torrust-demo.com;
+
+    error_log /var/log/nginx/error.log debug;
+
+    ...
+}
+```
+
+</CodeBlock>
+
+If your server does not have too much disk space you could run out of disk space.
+
+Docker stores the logs in a separate directory, if you wan to truncate the logs you can find them first with:
```

Comment:
if you want to truncate

you forgot a "t"

Reply:
Fixed